Lanchonete/LanchoneteTests/Managers/ShoppingCartManagerTests.swift

```diff
@@ -13,33 +13,26 @@ class ShoppingCartManagerTests: XCTestCase {
     func test_GIVEN_an_empty_shopping_cart_WHEN_I_add_a_sandwich_THEN_the_total_should_be_the_price_of_the_added_sandwich() {
         let manager = ShoppingCartManager()
         
-        manager.addSandwich(type: .xBacon)
+        manager.addSandwichWithQuantity(sandwich: .xBacon, quantity: 1, price: 7.0)
```

Comment:
`addSandwichWithQuantity` has two flows, one for a new item (tested here) and another for an existing item that is not being tested.

Reply:
I'll add the test now